### PR TITLE
fix depwarns

### PR DESCRIPTION
**Pull request by SimonDanisch**
WinRPM is pretty much unusable right now, since there are so many depwarns. 
I'm not 100% sure if this works on 0.3 and 0.4... I hope travis will catch anything fishy ;)

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -72,12 +72,12 @@ println("PASSED 12.1")
 @test (LibExpat.find(pd, "/ListBucketResult/Contents[2]/Owner/JUNK#string") == nothing)
 println("PASSED 12.2")
 
-pd = xp_parse(open(readall, joinpath(DATADIR,"utf8.xml")))
+pd = xp_parse(open(readstring, joinpath(DATADIR,"String.xml")))
```
Comment (tkelman) [inline test/runtests.jl]: you didn't rename the xml file did you?

Comment (SimonDanisch) [inline test/runtests.jl]: woops, the dangers of find & replace...
Fixed the test in new commit!

**Comment by SimonDanisch** [comment]
Rebased and ready to merge, I suppose ;)

File: REQUIRE [hunk 2]
```diff
@@ -1,2 +1,2 @@
 julia 0.3
-Compat 0.7.1
+Compat 0.7.15
```
Comment (tkelman) [inline REQUIRE]: wasn't `unsafe_string` newer than this?

Comment (SimonDanisch) [inline REQUIRE]: yupp!

**Pull request merged**